Tools-Override/codeAnalysis.targets

```diff
@@ -19,8 +19,8 @@
     <UseWin32Apis Condition="'$(OSGroup)'=='Windows_NT'">true</UseWin32Apis>
```

Comment:
I do not know what UseWin32APIs=false means. We only enter this block if it's Windows_NT in the first place. So effectively we only use two of these files. To clean up later.